specs/chrome_extensions.table

```diff
@@ -1,21 +1,31 @@
 table_name("chrome_extensions")
-description("Chrome browser extensions.")
+description("Chrome-based browser extensions.")
 schema([
-    Column("uid", BIGINT, "The local user that owns the extension",
-        index=True),
+    Column("browser_type", TEXT, "The browser type (Valid values: chrome, chromium, opera, yandex, brave, edge, edge_beta)"),
+    Column("uid", BIGINT, "The local user that owns the extension", index=True),
     Column("name", TEXT, "Extension display name"),
-    Column("profile", TEXT, "The Chrome profile that contains this extension"),
-    Column("identifier", TEXT, "Extension identifier"),
+    Column("profile", TEXT, "The name of the Chrome profile that contains this extension"),
+    Column("profile_path", TEXT, "The profile path"),
+    Column("identifier", TEXT, "Extension identifier (folder name)"),
     Column("version", TEXT, "Extension-supplied version"),
     Column("description", TEXT, "Extension-optional description"),
-    Column("locale", TEXT, "Default locale supported by extension"),
+    Column("default_locale", TEXT, "Default locale supported by extension", aliases=["locale"]),
+    Column("current_locale", TEXT, "Current locale supported by extension"),
     Column("update_url", TEXT, "Extension-supplied update URI"),
     Column("author", TEXT, "Optional extension author"),
-    Column("persistent", INTEGER,
-        "1 If extension is persistent across all tabs else 0"),
+    Column("persistent", INTEGER, "1 If extension is persistent across all tabs else 0"),
     Column("path", TEXT, "Path to extension folder"),
     Column("permissions", TEXT, "The permissions required by the extension"),
+    Column("permissions_json", TEXT, "The JSON-encoded permissions required by the extension", hidden=True),
     Column("optional_permissions", TEXT, "The permissions optionally required by the extensions"),
+    Column("optional_permissions_json", TEXT, "The JSON-encoded permissions optionally required by the extensions", hidden=True),
+    Column("manifest_hash", TEXT, "The SHA256 hash of the manifest.json file"),
+    Column("referenced", BIGINT, "1 if this extension is referenced by the Preferences file of the profile"),
+    Column("from_webstore", TEXT, "True if this extension was installed from the web store"),
+    Column("state", TEXT, "1 if this extension is enabled"),
+    Column("install_time", TEXT, "Extension install time, in its original Webkit format"),
```

Comment:
Is there value in preserving this? (I think most of the windows tables just convert, and return the converted value)

Reply:
The Webkit format is pretty strange, so it's not immediate to convert it to/from unix timestamps. The reason I kept it there is because it lets users look for timestamps they may have seen in their local files